hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -64,6 +64,8 @@
       "dfs.container.ratis.ipc";
   public static final int DFS_CONTAINER_RATIS_IPC_PORT_DEFAULT = 9858;
 
+  public static final String OZONE_HTTP_POLICY = "ozone.http.policy";
+
```

Comment:
This newly added property should be used during HTTP server start. 
That part is missing.